doc/asab/web/restapidocs.rst

```diff
@@ -0,0 +1,107 @@
+REST API Docs
+==============
+
+ASAB's API service generates a `Swagger documentation <https://swagger.io/specification>`_ which automatically shows all
+of your endpoints. However you will need to follow some steps to add summaries, tags and more.
```

Comment:
"Moreover, you can add summaries, tags, and more."
Sounds more optimistic, doesn't it?